src/boot/store.js

```diff
@@ -14,17 +15,20 @@ import middleware from './middleware';
 //   rootReducer,
 //   compose(autoRehydrate(), applyMiddleware(...middleware)),
 // );
+const reduxPersistConfigV4 = {
+  whitelist: [...config.storeKeys, ...config.cacheKeys],
```

Comment:
Extract to `const whitelist = [...config.storeKeys, ...config.cacheKeys]` and then reuse in both `reduxPersistConfigV4` and `reduxPersistConfig`.